font/src/ft/mod.rs

```diff
@@ -108,7 +108,7 @@ impl Rasterize for FreeTypeRasterizer {
 
         // Fallback for bitmap fonts which do not provide underline metrics
         if underline_position == 0. {
-            underline_thickness = (descent / 5.).round();
+            underline_thickness = (descent.abs() / 5.).round();
```

Comment:
I don't think `.abs()` is useful here, the descent should always be negative according to freetype docs. Have you found an example where this is not the case?

Reply:
Their docs aren't clear. Read the "descender" thing.


>The descender is the vertical distance from the horizontal baseline to the lowest ‘character’ coordinate in a font face. Unfortunately, font formats don't define the descender in a uniform way. For some formats, it represents the descent of all capital latin characters (without accents), for others it is the ascent of the lowest accented character, and finally, other formats define it as being equal to bbox.yMin. This field is negative for values below the baseline.


As you can see, it does state "always negative".